site/js/main.js

```diff
@@ -57,6 +58,13 @@ window.addEventListener('DOMContentLoaded', async () => {
     offlineSearch.style.display = 'block';
   }
 
+  if (body) {
+    const { Theme } = await import('./worklets/theme');
+    const theme = new Theme(body);
+    const storedTheme = theme._getStoredTheme();
+    theme.name = storedTheme;
+  }
```

Comment:
A couple of things here:
* Appreciate the re-use of our conditional checking pattern here, but `body` is always going to be available, so we don't need to add a conditional in for it. We also don't need to pass it in; it's a known entity that we can grab directly (aka doesn't require querying for)
* I don't see `storedTheme` or `theme` being used outside of this initialization. If they're not, we can just do `new Theme()`